server/src/resolversImpl.ts

```diff
@@ -93,11 +94,30 @@ export async function discoverCluster(
   context.logger.debug(`Fetching flows for discovery...`);
   const flowsConnection = await context.database.getFlows(
     context,
+    { filterBy },
     {
-      filterBy
-    },
-    1000000,
-    true
+      processL7: true,
+      flowsFiltersExtensionCallback: ({
+        srcBlacklistFilter,
+        dstBlacklistFilter
+      }) => {
+        srcBlacklistFilter.addVerdict(Verdict.DROPPED);
```

Comment:
shouldn't we keep dropped flows?

Reply:
for Service Map we are filtering out Dropped flows and don't use them in service discovery for now.

Reply:
ok